src/renderer/helpers/envHelper.ts

```diff
@@ -11,3 +11,11 @@ export const isEnv = (env: ENV | undefined): env is ENV => !!env
  * @param defaultValue {string} Default value
  */
 export const envOrDefault = (env: ENV | undefined, defaultValue: string) => (isEnv(env) ? env : defaultValue)
+
+/**
+ * Read enabled chains from a given ENV.
+ * @param env {string} ENV
+ */
+export const loadEnabledChains = (env: ENV | undefined) => {
+  return envOrDefault(env, 'THOR,BNB,BTC,ETH').split(',')
```

Comment:
I would suggest to use `THOR,BNB,BTC` as default, so we can use `REACT_APP_CHAINS_ENABLED` for development only. 

For production, `REACT_APP_CHAINS_ENABLED` don't need to be set. That's whenever we release a new chain, we have to update it here in code, but not in [ Github SECRETS of ASGDX](https://github.com/thorchain/asgardex-electron/settings/secrets/actions)

Reply:
Make sense